data/human/deep missions.txt

```diff
@@ -2752,3 +2752,1877 @@ mission "Stone of our Fathers 5"
+			`	"Captain <last>, it is a pleasure to hear from you again. What-so-ever can I help you with?" She says with believable cheerfulness.`	
+			`	You tell her about the Skadenga hiding in the spaceport and their plans to escape to Nifel.`
+			`	"Oh dear!" Carla says. "How terrible. Those poor foolish savages would surely suffer on that terrible frozen world. How misguided of them. We are aware that some have struggled to assimilate. It's quite tragic, but we will handle things from here. Hopefully, we can convince them to integrate eventually, it is really for the best. I wish their old leader could have seen that before she died. It would have been easier for everyone."`
+			`	"As always, we appreciate your valuable contributions. I'll authorize a generous sum of credits to be transferred to your account. I need to get some approvals so it may take a couple of days to come through. Thank you, and buh-bye."`
+				decline
+			label Hjlod
+			`	You return to Hjlod, who is surprised that you came back at all, and let her know that you going to help them.`
+			label help
+			`	Hearing this, the slumped and depressed dredges in the hall stir and rise from their stupor. Hurriedly some gather items while others race off further into the dark unused parts of the spaceport.`
+			`	"Ve have 52 people that need off this vorld right now."`
+			`	Hjlod hesitates a moment and then adds. "I know how much you like money. Ve have pooled all our savings together and will pay you 12,481 credits for this job."`
+			`	Hjlod grabs a raggedy duffle bag and a group gather around her. "The rest vill meet us at your ship. Ve move now."`
+			choice
+				`	(Follow Hjlod.)`
+			`	Once you're back in the operational spaceport you and your group move quickly, while drawing occasional glances from curious on-lookers. A large group of disheveled Skaldenga are waiting for you when you reach your ship.`
+			`	"Everyone on, quickly!" Hjlod barks at her people.`
+			`	A few moments later they are all on board. "Asdis, headcount!" Hjlod says.`
+			`	As you're waiting two more dirty and gaunt Skadenga show up. Asdis adds them to her count and then addresses Hjlod. "Ondurdis, 48 total."`
+			`	Hjlod frowns. "It's Arnson's family isn't it? I told him he was too far."`
+			`	A commotion boils up from a few terminals down, and you see some civil servants and local police officers heading your way.`
+			`	Asdis is also watching. "They could still be coming."`
+			`	"It's too late for them." Hjlod shuts the hatch. "<first>, if ever ve vere friends. Take off, now."`
+			choice
+				`	(Take off.)`
+					goto takeoff
+				`	(Power down your ship and open the hatch.)`
+			`	You turn off your ship's ignition, pop open the hatch before Hjlod can react, and jump out. You find Carla Von Hausen, and a squad of police officers waiting for you.`
```

Comment:
If they were already there by the time you open the hatch then there should've been some warning just before - just squeezes a few extra seconds into the sequence of events.
```suggestion
			`	You turn off your ship's ignition, pop open the hatch before Hjlod can react, and jump out. Carla Von Hausen, at least two civil service works, and a squad of police officers arrive within seconds.`
```